pkg/kubelet/api/v1alpha1/stats/types.go

```diff
@@ -119,9 +119,9 @@ type CPUStats struct {
 	Time unversioned.Time `json:"time"`
 	// Total CPU usage (sum of all cores) averaged over the sample window.
 	// The "core" unit can be interpreted as CPU core-nanoseconds per second.
-	UsageNanoCores *uint64 `json:"usageNanoCores,omitempty"`
+	UsageRate *uint64 `json:"usageNanoCores,omitempty"`
 	// Cumulative CPU usage (sum of all cores) since object creation.
-	UsageCoreNanoSeconds *uint64 `json:"usageCoreNanoSeconds,omitempty"`
+	CumulativeNanoseconds *uint64 `json:"usageCoreNanoSeconds,omitempty"`
```

Comment:
I'd like to keep cores in the units here to make it clear what we're measuring (`UsageRateCores` and `CumulativeCoreNanoseconds`)